packages/did-key-web-crypto/src/functions/publicKeyToFingerprint.ts

```diff
@@ -0,0 +1,17 @@
+import bs58 from 'bs58';
+
+import { crv_to_multicodec } from '../constants';
+import { jwkToBase58 } from './jwkToBase58';
+
+export const publicKeyToFingerprint = (publicKeyJwk: any): string => {
```

Comment:
@msporny @dmitrizagidulin @dlongley If you have a spare cycle to look at this, I would love a second eye on it.

I am attempting to generalize support for did key representations of anything that can be represented via a JWK.